The plugin fails if target is not already created

If this plugin is the first to require the target directory and target is not already created by another plugin then this plugin fails with

```
> [ERROR] TRUSTSTORE: Error while generating truststore: /Users/David/Documents/projects/gateway.truststore/target/truststore.db (No such file or directory)
```

--- a/src/main/java/org/kaazing/maven/plugins/TrustStoreMojo.java
+++ b/src/main/java/org/kaazing/maven/plugins/TrustStoreMojo.java
@@ -259,7 +259,11 @@ public class TrustStoreMojo
             Map<String, String> certs = getCertificates(trustStoreSourceURL);
             KeyStore ks = getTrustStore(certs, trustStoreType);
 
-            tmpFile = new File(trustStoreTmpFile);
+            tmpFile = new File(project.getBuild().getDirectory());
+
+            // make sure that the target directory exists:
+            new File(tmpFile.getParent()).mkdirs();
+
             fos = new FileOutputStream(tmpFile);
             ks.store(fos, trustStorePass.toCharArray());
 

--- a/src/main/java/org/kaazing/maven/plugins/TrustStoreMojo.java
+++ b/src/main/java/org/kaazing/maven/plugins/TrustStoreMojo.java
@@ -259,7 +259,7 @@ public class TrustStoreMojo
             Map<String, String> certs = getCertificates(trustStoreSourceURL);
             KeyStore ks = getTrustStore(certs, trustStoreType);
 
-            tmpFile = new File(project.getBuild().getDirectory());
+            tmpFile = new File(trustStoreTmpFile);
 
             // make sure that the target directory exists:
             new File(tmpFile.getParent()).mkdirs();

